CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#6207](https://github.com/thanos-io/thanos/pull/6207) Receive: Remove the shipper once a tenant has been pruned.
 - [#6216](https://github.com/thanos-io/thanos/pull/6216) Receiver: removed hard-coded value of EnableExemplarStorage flag and set it according to max-exemplar value.
 - [#6222](https://github.com/thanos-io/thanos/pull/6222) mixin(Receive): Fix tenant series received charts.
+- [#6218](https://github.com/thanos-io/thanos/pull/6218) Store: handle ResourceExhausted as a non-server error.
```

Comment:
Should we specify it's mixin? I don't know if we have a convention on how to identify mixin changes (but e.g. one line above we put mixin in the changelog), but currently it sounds this could be a code change in store.

Reply:
Makes sense. I'll update it 👍

Reply:
Done.